file_root/top.sls

```diff
@@ -2,7 +2,7 @@
 
 openstack:
   "*":
-    - generics.*
+    - generics.prerequisites
```

Comment:
E un motiv pt specificat direct generics.prereq? Altfel ar putea intra in for %formula%, sau de ce s-a scos generics.*?